src/components/EventCard/index.jsx

```diff
@@ -0,0 +1,166 @@
+import { Box, Typography } from '@material-ui/core'
+import { LocationOn } from '@material-ui/icons'
+import { makeStyles } from '@material-ui/styles'
+import moment from 'moment'
+import ClampLines from 'react-clamp-lines'
+import PropTypes from 'prop-types'
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    boxShadow: '0px 10px 13px -7px #efefef, 5px 5px 15px 5px rgba(0,0,0,0)',
+    width: '20em',
+    padding: '1em',
+    transform: 'translateY(0px);',
+    transition: '0.5s',
+    marginTop: '20px',
+    borderRadius: '1.5em',
+
+    [theme.breakpoints.down('xs')]: {
+      width: '100%',
+    },
+    '&:hover': {
+      boxShadow: '0px 10px 13px -7px #999999, 5px 5px 15px 5px rgba(0,0,0,0)',
+
+      transform: 'translateY(-10px);',
+    },
+  },
+  cardHeader: {
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'center',
+    gap: '0.5em',
+
+    [theme.breakpoints.down('xs')]: {
+      flexDirection: 'column',
+      alignItems: 'flex-start',
+      '& > div > div > h6': {
+        marginTop: '0.3em',
+      },
+    },
+  },
+  dateBox: {
+    backgroundColor: '#FAE9D0',
```

Comment:
For all CSS color hex codes, can we reuse them from the palette in /theme folder? If the hex code is not there, can add them in too.